storage/indexer/src/db_ops.rs

```diff
@@ -17,6 +17,15 @@ pub fn open_db<P: AsRef<Path>>(db_path: P, rocksdb_config: &RocksdbConfig) -> Re
     )?)
 }
 
+pub fn open_tailer_db<P: AsRef<Path>>(db_path: P, rocksdb_config: &RocksdbConfig) -> Result<DB> {
+    Ok(DB::open(
+        db_path,
+        "tailer_db",
```

Comment:
Shall we use a name that can relate to this better? https://github.com/aptos-labs/aptos-core/blob/625118a03f8f49e36c97ed709080a7ddccbe7484/ecosystem/indexer-grpc/indexer-grpc-table-info/src/runtime.rs#L14

index_indexer_db2 works better than `tailer_db` 😂 , wdyt?

(also how about defining a constant like we do everywhere else?)